### DISCUSSION TITLE
Fresh Future

### DISCUSSION THREAD

**Discussion by thegaryroberts** [freshframework/fresh#190] [Q&A] [answered]
Is there a Roadmap for Fresh?

Currently there is a big DO NOT USE warning on the Readme! But so far it is the only Deno first Web SPA/SSR framework I've managed to get reasonably working (and I have tried a lot of options!). The documentation is coming together well, and you have a decent approach to testing taking shape. So basically I want to use this (please let me).

**Comment by lucacasonato** [comment, marked as answer]
I will cut the first real release within the next couple of weeks here. It's getting really close, I just need to iron out some final DX and UX issues.
